app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -30,7 +31,8 @@ class HomeFragmentPresenter @Inject constructor(
 ) {
 
   private val routeToExplorationListener = activity as RouteToExplorationListener
-
+  private val routeToTopicListener = activity as RouteToTopicListener
+  private val TAG_HOME_FRAGMENT = "HomeFragment"
```

Comment:
Declare this in similar way `EXPLORATION_ID` has been declared, on top of the class.